### DISCUSSION TITLE
🚀 OIDC connector

### DISCUSSION THREAD

**Discussion by ClaudioNTT** [dexidp/dex#3659] [General]
I noticed from the doc that the OIDC connector is in beta. Since the OIDC protocol is used for inbound requests to Dex, I didn't expect to see the connector in beta. Is there a specific reason for this? Are there any details to consider to understand if it should be taken into consideration? Thanks so much!

**Comment by nabokihms** [comment]
It is stable, we must change the doc. Thanks.
